src/clusterfuzz/_internal/google_cloud_utils/storage.py

```diff
@@ -1170,3 +1200,89 @@ def get_signed_download_url(remote_path, minutes=SIGNED_URL_EXPIRATION_MINUTES):
   contents."""
   provider = _provider()
   return provider.sign_download_url(remote_path, minutes=minutes)
+
+
+def _error_tolerant_download_signed_url_to_file(url, path):
+  try:
+    return download_signed_url_to_file(url, path), url
+  except Exception:
+    return None
+
+
+def _error_tolerant_upload_signed_url(url, path):
```

Comment:
Maybe I'm missing this in this large PR, but who calls this?

Reply:
upload_signed_urls. We don't want to kill the uploading process if one thread exceptions.